frontend/src/components/organisms/MatchesList.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from "styled-components";
+import {Card} from '@material-ui/core';
+import {MatchItem} from "../molecules/MatchItem/MatchItem";
+import {MIN_CARD_COMPONENT_WIDTH} from "../atoms/constants/sizes";
+
+const exampleDataBase: MatchListProps[] = [
+    {
+        onClickTeam: callBackFunction,
+        matchNumber: 0,
+        level: 1,
+        matchStatus: "aktywny",
+        team1: {
+            player1: "Lord Json",
```

Comment:
![image](https://user-images.githubusercontent.com/18637116/113932062-7c1c2b00-97f3-11eb-8a1c-2e902fab55de.png)
Nie wiem co masz w glowie, ale mi sie to podoba :) Mozesz uzyc MSW do takiego mockowania.

Reply:
Użyję przy następnym PR, w tym za bardzo się rozpisałem i chciałbym już zmergować - bo mocno stresują mnie kolejne zmiany na developie